master/meta_node.go

```diff
@@ -48,15 +49,18 @@ type MetaNode struct {
 	PersistenceMetaPartitions []uint64
 	RdOnly                    bool
 	MigrateLock               sync.RWMutex
+	CpuUsed                   atomicutil.Float64 `json:"-"`
 }
 
 func newMetaNode(addr, zoneName, clusterID string) (node *MetaNode) {
-	return &MetaNode{
+	node = &MetaNode{
 		Addr:     addr,
 		ZoneName: zoneName,
 		Sender:   newAdminTaskManager(addr, clusterID),
 		Carry:    rand.Float64(),
 	}
+	node.CpuUsed.Store(0)
```

Comment:
return &MetaNode{
		Addr:     addr,
		ZoneName: zoneName,
		Sender:   newAdminTaskManager(addr, clusterID),
		Carry:    rand.Float64(),
               CpuUtil 0,
	}

Reply:
`node.CpuUsed.Store(0)` is better than `CpuUtil:0`, because there is a `uint64` in low level of `atomicutil.Float64`.

`node.CpuUsed.Store(0)` is more readable.